envs/centos7/py39/requirements.txt

```diff
@@ -0,0 +1,110 @@
+alabaster==0.7.12
+anyio==2.2.0
+arcgis==1.8.2
+argon2-cffi==20.1.0
+async-generator==1.10
+attrs==20.3.0
+Babel==2.9.1
+backcall==0.2.0
+bleach==3.3.0
+certifi==2020.12.5
+cffi==1.14.5
+chardet==4.0.0
+colorama==0.4.4
+cryptography==3.4.7
+cycler==0.10.0
+decorator==5.0.7
+defusedxml==0.7.1
+deprecation==2.1.0
+docutils==0.16
+entrypoints==0.3
+idna==2.10
+imagesize==1.2.0
+importlib-metadata==4.0.1
+ipykernel==5.5.3
+ipython==7.23.0
+ipython-genutils==0.2.0
+ipywidgets==7.6.3
+jedi==0.18.0
+jeepney==0.6.0
+Jinja2==2.11.3
+json5==0.9.5
+jsonschema==3.2.0
+jupyter-client==6.1.12
+jupyter-core==4.7.1
+jupyter-packaging==0.9.2
+jupyter-server==1.6.4
+jupyterlab==3.0.14
+jupyterlab-pygments==0.1.2
+jupyterlab-server==2.5.0
+jupyterlab-widgets==1.0.0
+keyring==23.0.1
+kiwisolver==1.3.1
+lerc==0.1.0
+MarkupSafe==1.1.1
+matplotlib==3.4.1
+matplotlib-inline==0.1.2
+mistune==0.8.4
+nbclassic==0.2.7
+nbclient==0.5.3
+nbconvert==6.0.7
+nbformat==5.1.3
+nest-asyncio==1.5.1
+notebook==6.3.0
+ntlm-auth==1.5.0
+numpy==1.20.2
+oauthlib==3.1.0
+packaging==20.9
+pandas==1.2.4
+pandocfilters==1.4.3
+parso==0.8.2
+pexpect==4.8.0
+pickleshare==0.7.5
+Pillow==8.2.0
+pkginfo==1.7.0
+prometheus-client==0.10.1
+prompt-toolkit==3.0.18
+ptyprocess==0.7.0
+pycparser==2.20
+Pygments==2.9.0
+pyparsing==2.4.7
+pyrsistent==0.17.3
+pyshp==2.1.3
+python-certifi-win32==1.6
+python-dateutil==2.8.1
+pytz==2021.1
+pyzmq==22.0.3
+readme-renderer==29.0
+requests==2.25.1
+requests-ntlm==1.1.0
+requests-oauthlib==1.3.0
+requests-toolbelt==0.9.1
+rfc3986==1.4.0
+SecretStorage==3.3.1
+Send2Trash==1.5.0
+setuptools-scm==6.0.1
+six==1.15.0
+sniffio==1.2.0
+snowballstemmer==2.1.0
+Sphinx==3.5.4
+sphinxcontrib-applehelp==1.0.2
+sphinxcontrib-devhelp==1.0.2
+sphinxcontrib-htmlhelp==1.0.3
+sphinxcontrib-jsmath==1.0.1
+sphinxcontrib-qthelp==1.0.3
+sphinxcontrib-serializinghtml==1.1.4
+terminado==0.9.4
+testpath==0.4.4
+tomlkit==0.7.0
+tornado==6.1
+tqdm==4.60.0
+traitlets==5.0.5
+twine==3.4.1
+ujson==4.0.2
+urllib3==1.26.4
```

Comment:
Based on the security analysis performed by dependabot, urllib3 needs to be updated to version 1.26.5.

Reply:
done